master.go

```diff
@@ -105,21 +107,45 @@ func (m *Master) ScheduleFromId(ID string) error {
 	return nil
 }
 
+func (m *Master) logScheduleInfo(message string, payload *MetreMessage) {
+	log.Info(fmt.Sprintf("metre.master.Schedule: %s - taskId:%s, uid:%s, jobGUID:%s",
+		message,
+		payload.TaskId,
+		payload.UID,
+		payload.JobGUID))
+}
+
+func (m *Master) logScheduleErr(message string, payload *MetreMessage) {
+	log.Error(fmt.Sprintf("metre.master.Schedule: %s - taskId:%s, uid:%s, jobGUID:%s",
+		message,
+		payload.TaskId,
+		payload.UID,
+		payload.JobGUID))
+}
+
 // FIXME : Scheduling happens here. Is this the right design ?
 func (m *Master) Schedule(taskId string, uid string) (string, error) {
-	msg := CreateMsg(Request, taskId, uid, "")
-	req := NewScheduleInput(msg)
+	// jobGUID is a low-level tracking guid at the metre level.  Tasks can also have their
+	// own UID for tracking which is specific to the app.
+	jobGUID := uuid.NewV4().String()
+
```

Comment:
Right now there is jobGUID for each `schedule`. So if a task has `n` records to process.  

If bufferLength is set to `k` for each `schedule` then we will still have `n/k` jobGUID's in the logs to track. 

But I think it's great to have it.